### PR TITLE
Set up Mongoose and Created User Schema

### PR DESCRIPTION
**Pull request by melvyn9**
1. Connected to the mongoDB server using mongoose. Removed database.js as it was not needed.
2. Created user schema.

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        required: true
+    },
+    age: {
+        type: Number,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    }
```
Comment (shad674) [inline models/user.js:27]: This is a good start! We will add more schema in the future in terms of the people they like to be room with!

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema; // Used to create a new Schema
+
+// Define the Schema for user
+// TODO we can check for validation in the future
```
Comment (shad674) [inline models/user.js:6]: For validation and the next CR can you connect one of the GET endpoint to get any data from the database

**Review by shad674 — APPROVED** [review]

**Review by sammg13 — APPROVED** [review]

**Review by sammg13 — APPROVED** [review]

File: models/user.js (hunk 2 above)
Comment (melvyn9) [reply models/user.js:6]: Resolved

File: controllers/admin.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const User = require('../models/user');
+
+// Used in routes/index.js. Adds data into the database when POSTing to /add-product.
+exports.addUser = (req, res, next) => {
+    // Obtain data and store it.
+    const firstName = req.body.firstName;
+    const lastName = req.body.lastName;
```
Comment (shad674) [inline controllers/admin.js:7]: Nice

File: server.js [hunk 4]
```diff
@@ -1,15 +1,57 @@
...
 
-app.get("/", (req, res) => {
-  res.send("Hello Shad");
-});
+// Registering Routes
+const mainRoutes = require('./routes/index');
+app.use(mainRoutes);
+
+// Mongoose
+const mongoose = require('mongoose');
```
Comment (shad674) [inline server.js:11]: Nice formatting

**Review by shad674 — APPROVED** [review]
LGTM

**Pull request merged**